language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/SnippetGenerator.java

```diff
@@ -1040,7 +1040,7 @@ public static SnippetBlock getReturnStatementSnippet() {
     public static SnippetBlock getServiceDefSnippet() {
         ImmutablePair<String, String> httpImport = new ImmutablePair<>("ballerina", "http");
         String snippet = "service /${1} on new http:Listener(8080) {"
-                + CommonUtil.LINE_SEPARATOR + "\tresource function ${2:methodName} ${3:resourceName}"
+                + CommonUtil.LINE_SEPARATOR + "\tresource function ${2:get} ${3:getResource}"
```

Comment:
@nadeeshaan shall we move `getResource` into `resource` since repeating `get` looks bit weird. wdyt?

`resource function get getResource(...)` vs. `resource function get resource(...)`

Reply:
We can't use `resource` because it is a reserved keyword in the language

Reply:
Okay then shall we go forward with `resource function get httpResource(http:Caller caller, http:Request req)` ?